app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,23 @@
-<h2>Forgot your password?</h2>
+<div class="signup-view-container">
+  <div class="form">
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+    <%= link_to "X", root_path, class: "form-close-button" %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+    <h2 class="form-title">Forgot your password?</h2>
 
-  <div class="actions">
-    <%= f.submit "Send me reset password instructions" %>
-  </div>
-<% end %>
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :email, class: 'forgot_password_label' %>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "user@example.com", class: "input-form input-neutral" %>
+    </div>
 
-<%= render "devise/shared/links" %>
+    <div class="actions">
+      <%= f.submit "Send me reset password instructions", class: "submit-button forgot_password_button"%>
```

Comment:
Responsive problems in the button
![127 0 0 1_3000_users_password_new](https://github.com/BrightCoders-Institute/proyecto-final-bcmay23-ror-team1/assets/65278575/0b8199d6-b34b-49dc-806e-98f7ee801dfa)

Reply:
que chulada te quedó